src/dataviews/helpers/histogram-helper.js

```diff
@@ -2,31 +2,66 @@ var moment = require('moment');
 var _ = require('underscore');
 
 var MOMENT_AGGREGATIONS = {
-  day: 'd',
-  hour: 'h',
+  second: 's',
   minute: 'm',
+  hour: 'h',
+  day: 'd',
+  week: 'w',
   month: 'M',
   quarter: 'Q',
-  second: 's',
-  week: 'w',
   year: 'y'
 };
 
+function isLessThanDays (aggregation) {
+  return aggregation === 'second' || aggregation === 'minute' || aggregation === 'hour';
```

Comment:
I thought about it `isShorterThan` but since `MOMENT_AGGREGATIONS` is an object I'm not confident that the returned keys preserve always the order they were declared.

But thanks to your comment I see that the order returned by `Object.keys` is the same of `for..in` and MDN says:

```
The for...in statement iterates over the enumerable properties of an object, in original insertion order. 
```

So... it can be done.